databook/_toc.yml

```diff
@@ -11,79 +11,95 @@ chapters:
+      sections:
+      - file: physiology/ephys/cell-type-lookup-table/ctlut-session-data
+      - file: physiology/ephys/cell-type-lookup-table/ctlut-optotagging
+      - file: physiology/ephys/cell-type-lookup-table/ctlut-accessing-data
+      - file: physiology/ephys/cell-type-lookup-table/ctlut-identifying-tagged-units
+    - file: physiology/ephys/np-ultra/np-ultra                    # Also a placeholder that needs actual content
+  - file: physiology/stimuli/stimuli
     sections:
-    - file: visual-behavior/VBN-Dataset
-    - file: visual-behavior/VBN-SessionData
-    - file: visual-behavior/VBN-Tutorial-Aligning_Neural_Data_to_Stimuli.ipynb
-      title: Tutorial Aligning Neural Data to Stimuli
-    - file: visual-behavior/VBN-Tutorial-Aligning_Behavior_Data_to_Task_Events.ipynb
-      title: Tutorial Aligning Behavioral Data to Task Events
-    - file: visual-behavior/VBN-Tutorial-Analyzing_LFP_Data.ipynb
-      title: Tutorial Analyzing LFP data
-- file: microns-em/em-background
+    - file: physiology/stimuli/passive-visual-stimuli/visual-stimuli-list                    # Also a placeholder that needs actual content
+      sections:
+      - physiology/stimuli/passive-visual-stimuli/visual-stimuli-np
+    - file: physiology/stimuli/visual-behavior/VB-Behavior
+      title: Visual Behavior Task
+      sections:
+      - file: physiology/stimuli/visual-behavior/VB-BehaviorSessionData
+        title: Behavior Session Data
+  - file: physiology/pyNWB/pyNWB                    # Also a placeholder that needs actual content
+- file: anatomy/anatomy
   sections:
-  - file: microns-em/em-structural-data.md
-  - file: microns-em/proofreading.md
-  - file: microns-em/em-neuroglancer.md
-  - file: microns-em/em-dash-apps.md
-  - file: microns-em/em-caveclient-setup.md
-  - file: microns-em/em-querying-tables.md
-  - file: microns-em/key-tables.md
-  - file: microns-em/em-volume-data.md
-  - file: microns-em/em-skeletons.md
-  - file: microns-em/em-meshes.md
-  - file: microns-em/em-functional_data.md
-  - file: microns-em/em-coordinates.md
-  - file: microns-em/em-ultrastructure.md
+  - file: anatomy/microns-em/em-background
+    sections:
+    - file: anatomy/microns-em/em-structural-data.md
+    - file: anatomy/microns-em/proofreading.md
+    - file: anatomy/microns-em/em-neuroglancer.md
+    - file: anatomy/microns-em/em-dash-apps.md
+    - file: anatomy/microns-em/em-caveclient-setup.md
+    - file: anatomy/microns-em/em-querying-tables.md
+    - file: anatomy/microns-em/key-tables.md
+    - file: anatomy/microns-em/em-volume-data.md
+    - file: anatomy/microns-em/em-skeletons.md
+    - file: anatomy/microns-em/em-meshes.md
+    - file: anatomy/microns-em/em-functional_data.md
+    - file: anatomy/microns-em/em-coordinates.md
+    - file: anatomy/microns-em/em-ultrastructure.md
+  #- file: This will be something on Single Cell Whole Brain Anatomy once we've determined a name
```

Comment:
let's add the anatomy/single-cell-morphology section and it'll be a placeholder as we add that content